tests/sles4sap/hana_install.pm

```diff
@@ -15,23 +15,17 @@ use base 'sles4sap';
 use strict;
 use warnings;
 use testapi;
-use utils 'zypper_call';
+use utils qw(file_content_replace zypper_call);
 use version_utils 'is_sle';
 use POSIX 'ceil';
 
-sub upload_install_log {
-    script_run "tar -zcvf /tmp/hana_install.log.tgz /var/tmp/hdb*";
-    upload_logs "/tmp/hana_install.log.tgz";
-}
-
 sub run {
     my ($self) = @_;
     my ($proto, $path) = $self->fix_path(get_required_var('HANA'));
 
     my $sid      = get_required_var('INSTANCE_SID');
     my $instid   = get_required_var('INSTANCE_ID');
-    my $password = 'Qwerty_123';
-    set_var('PASSWORD', $password);
+    my $password = get_var('INSTANCE_PASSWORD', 'Qwerty_123');
```

Comment:
Since we're using this default password definition in several modules, WDYT about defining it as a variable in `lib/sles4sap` and then using the variable in this call to `get_var` and the others? It would prevent errors if someone decides to change the default password in one module but not in the others.

Reply:
Could be, but as I wrote in another comment I will check if the `PASSWORD` variable is really used in SLES4SAP tests.

Reply:
> But anyway a global `$password` variable is needed yes.

That's what I think. Just be careful with the name of the variable.